src/app/[slug]/[postSlug]/page.tsx

```
@@ -132,16 +133,28 @@ export default async function PrefixPostPage({
-    <PostLayout
-      post={post}
-      relatedPosts={relatedPosts}
-      seriesPosts={seriesPosts}
-      seriesTitle={seriesTitle}
-      prevPost={prev}
-      nextPost={next}
-      backlinks={backlinks}
-      slugRegistry={slugRegistry}
-    />
+    <>
+      <script type="application/ld+json" dangerouslySetInnerHTML={{ __html: serializeJsonLd(jsonLd) }} />
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)